juniper_official/Chassis/check-failover-configured.rule

```diff
@@ -0,0 +1,71 @@
+/*
+ * Checks if graceful-switchover is configured on a dual RE chassis or not
+ *
+ * First checks if other routing engine is present or not. If present, checks
+ * if failover (set chassis redundancy graceful-switchover) is configured or not.
+ * 
+ * In dual RE chassis, for high availability it is recommended to have faiover
+ * configured. A yellow color is reported if this configuration is not found
+ * in a dual Routing Engine chassis.
+ */
+iceberg {
+    topic chassis.gres {
+        rule check-failover-configured {
+            synopsis "Check whether graceful-switchover is configured or not";
+            description "Checks if other RE is present and if GRES/failover is configured or not";
+            sensor gres-sensor {
+                iAgent {
+                    file failover-info.yml;
+                    table getFailoverInfoTable;
+                    frequency 10m;
+                }
+            }
+            field gres-state {
+                sensor gres-sensor {
+                    path failover;
+                }
+                type integer;
+                description "GRES/failover configured state";
+            }
+            field ore-present {
+                sensor gres-sensor {
+                    path ore_present;
+                }
+                type integer;
+                description "Other RE present or not";
+            }
+            trigger report-failover-configured-state {
```

Comment:
1) Provide trigger description and synopsis detailing sensor info.
2) Trigger frequency is must as default frequency is 10s and trigger gets evaluated once in 10s. but here sensor frequecy is 10 mins hence trigger frequency can be greaterthan equal to 10 mins.
e.g. synopsis "Routing engine GRES kpi";
       description "sets health based on Routing engine CPU utilization for 15 minutes";